pinot-common/src/main/java/org/apache/pinot/common/datablock/MetadataBlock.java

```diff
@@ -76,15 +78,23 @@ public enum MetadataBlockType {
   static class Contents {
 
     private String _type;
+    private Map<String, String> _stats;
+
+//    @JsonCreator
+//    public Contents(@JsonProperty("type") String type) {
+//      _type = type;
+//    }
```

Comment:
why the removal of this?

Reply:
Error in jackson with two seperate constructors.